styles.css

```diff
@@ -0,0 +1,107 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+header {
+  position: absolute;
+  width: 100%;
+  height: 100vh;
+  left: 1px;
+  top: 1px;
+}
+
+.navbar {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  flex-direction: row;
+  padding: 10px;
+  margin: 0;
+  background-color: rgba(255, 255, 255, 0.459);
+}
+
+.logo {
+  font-family: Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  color: rgb(96, 112, 255);
+}
+
+#menu {
+  color: rgb(96, 112, 255);
+  font-size: 18px;
+  line-height: 20px;
+}
+
+.headline {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background-image: url("./img/pro.png");
+  background-size: cover;
+}
+
+.text-holder {
+  width: 80%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  padding-top: 100px;
+}
+
+.primary-text {
+  font-family: Arial, Helvetica, sans-serif;
```

Comment:
Kindly use the same font family as used in the Figma template throughout your application.

Reply:
I have tried to use that font family but the linter was giving me errors, what can you recommend doing?

Reply:
I have found that font in google fonts, by importing you mean that I should include a script tag referencing it in the head part of my html.